index.js

```diff
@@ -233,15 +238,22 @@ class K8sVMExecutor extends Executor {
     /**
      * Starts a k8s build
      * @method start
-     * @param  {Object}   config                A configuration object
-     * @param  {Object}   [config.annotations]  Set of key value pairs
-     * @param  {Integer}  config.buildId        ID for the build
-     * @param  {String}   config.container      Container for the build to run in
-     * @param  {String}   config.token          JWT for the Build
+     * @param  {Object}   config                    A configuration object
+     * @param  {Object}   [config.annotations]      Set of key value pairs
+     * @param  {Integer}  config.buildId            ID for the build
+     * @param  {Integer}  config.id                 pipelineId for the build
+     * @param  {Integer}  config.eventId            eventId for the build
+     * @param  {String}   config.container          Container for the build to run in
+     * @param  {String}   config.token              JWT for the Build
      * @return {Promise}
      */
     _start(config) {
-        const { buildId, container, token } = config;
+        const { buildId, jobId, eventId, container, token } = config;
+        let pipelineId = '';
+
+        if (config.pipeline) {
+            pipelineId = config.pipeline.id || '';
+        }
```

Comment:
Minor Comment: Declaration and assignment can be combined!

Reply:
fixed